docs/install/containers.md

```diff
@@ -227,7 +227,8 @@ To upgrade Harvest:
 2. [Stop all containers](#stop-all-containers)
 
 3. Regenerate your `harvest-compose.yml` file by
-   running [harvest generate](#generate-a-docker-compose-for-your-pollers)
+   running [harvest generate](#generate-a-docker-compose-for-your-pollers).
+   This step is essential as it updates local copies of templates and dashboards, which are then mounted to the containers
```

Comment:
Make sure you don't skip this step. It is essential as it updates local copies of templates and dashboards, which are then mounted to the containers. If this step is skipped, Harvest will run with older templates and dashboards which will likely cause problems.